collection-scripts/gather_network_observability

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+BASE_COLLECTION_PATH="must-gather"
+NO_NS="$(oc get subs -A -o template --template '{{range .items}}{{if eq .spec.name "netobserv-operator"}}{{.metadata.namespace}}{{end}}{{end}}')"
+
+if [ -z "${NO_NS}" ]; then
```

Comment:
This to me (while perfectly fine); makes me wonder if this should be part of the main must-gather collection; or if this whole collection should be part of a NetObsever collector image. 

Last I checked the NetObserver Operator is not part of the core payload; thus it has its own life cycle and release cadence; meaning that if bugs are found; that team may want to fix them (out of band of the OpenShift release cadance and cycle).  Meaning that this approach to merge the collection of thing into the main collector isn't the right approach.